manager/src/main/java/io/gingersnapproject/database/vendor/Vendor.java

```diff
@@ -53,4 +54,8 @@ static PrimaryKey extractPrimaryKeyInLowerCase(RowSet<Row> rows) {
       }
       return new PrimaryKey(pkName, pkColumns);
    }
+
+   default String whereClause(List<String> keys) {
```

Comment:
I'm not sure a `default` implementation makes sense given that the placeholders are different for every DB vendor.

Related, could you also add the implementations for Oracle and Postgres :angel:

Reply:
oh sure, I thought mssql was the only one with different placeholder

Reply:
Sadly I already did this in https://github.com/gingersnap-project/cache-manager/pull/49/files#diff-0fd5204bef5db21cd685c49abd1953b797863596f39ca8e8f95fb349e78d2baaR16. I had done it where each vendor only overrides a method to define the parameter with position.